### PR TITLE
Add basic plugin for tmux

### PR DESCRIPTION
**Pull request by aphecetche**

### CODE REVIEW

File: plugins/tmux [hunk 1]
```diff
@@ -0,0 +1 @@
+repository = https://github.com/aphecetche/asdf-tmux
```
Comment (Stratus3D) [inline plugins/tmux]: This URL should be the URL of the Git repository, which in this case would be `https://github.com/aphecetche/asdf-tmux.git`.

**Comment by Stratus3D** [comment]
Other than the Git repo URL this looks good to me.

File: plugins/tmux (hunk 1 above)
Comment (aphecetche) [reply plugins/tmux]: Done.

**Comment by Stratus3D** [comment]
Thanks for the PR!

**Pull request merged**